test/e2e/storage/persistent_volumes-local.go

```diff
@@ -356,6 +360,97 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("should support SELinux labeling on local storage volume", func() {
+
+		var testVol *localTestVolume
+
+		BeforeEach(func() {
+			testVol = setupLocalVolumePVCPV(config, node0)
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+		})
+
+		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2 when using the same SELinux label", func() {
+			By("Creating podSc1")
+			podSc1, podSc1Err := createSecPod(config, testVol, false, false, &v1.SELinuxOptions{
+				Level: "s0:c0,c1",
+			})
+			Expect(podSc1Err).NotTo(HaveOccurred())
+
+			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, podSc1))
+			podSc1NodeName, podSc1NodeNameErr := podNodeName(config, podSc1)
```

Comment:
Sure, then we should only use new version of makeSecPod which build Security aware pods. I can add const with default SELinux labels to use them with all pods and SELinux specific cases will use non default labels.